File: androidApp/src/main/java/com/medhelp/medhelp/data/model/ServiceList.java
//package com.medhelp.medhelp.data.model;
//
//import com.google.gson.annotations.SerializedName;
//
//import java.util.ArrayList;
//import java.util.List;
//
//@SuppressWarnings("unused")
//public class ServiceList {
//
//    @SerializedName("imgError")
//    private boolean error;
//
//    @SerializedName("Message")
//    private String message;
//
//    @SerializedName("response")
//    private List<ServiceResponse> services = new ArrayList<>();
//
//    public String getMessage() {
//        return message;
//    }
//
//    public void setMessage(String message) {
//        this.message = message;
//    }
//
//    public boolean isError() {
//        return error;
//    }
//
//    public void setError(boolean error) {
//        this.error = error;
//    }
//
//    public List<ServiceResponse> getServices() {
//        return services;
//    }
//
//    public void setServices(List<ServiceResponse> services) {
//        this.services = services;
//    }
//}

File: shared/src/commonMain/kotlin/com/medhelp/newmedhelp/model/ServiceList.kt
package com.medhelp.newmedhelp.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class ServiceList {
    @SerialName("error")
    var error = false

    @SerialName("message")
    var message: String? = null

    @SerialName("response")
    var services: List<ServiceResponse>? = null
}